priv/riak_kv.schema

```diff
@@ -121,8 +121,7 @@
 %% faster by doubling.  There are 1M segments in a standard tree overall.
 {mapping, "tictacaae_maxresults", "riak_kv.tictacaae_maxresults", [
   {datatype, integer},
-  {default, 128},
-  hidden
+  {default, 64}
```

Comment:
But need update in PR explanation as well

`What is implemented here is prompted repairs. There is as before, scheduled exchanges, however these by default only repair 128 segments not 256 (to reduce overheads). These exchanges are otherwise unchanged.`

This is 64 now, right? and if so, then do you need to re-explain :
`These loops will have the tictacaae_maxresults boosted by a tictacaae_rangeboost integer (default 2 i.e. go to 256).`
I would expect the default now to be 4???

Reply:
The change in constants happened after the [volume test above](https://github.com/basho/riak_kv/pull/1806#issuecomment-975360319).

With maxresults=128 and rangeboost=2, the repair was faster, but it used more CPU than the current solution.

With maxresults=64 and rangeboost=2, the repair was still better than as-is (though not as goos as with 128), but the CPU use was the same as-is.

It was felt it was safer to make the introduction by default in such a way that it wouldn't use more resource, and leave it up to the user if they want to tune it higher and accept the trade off of  further improved performance for further resource utilisation.